dlrover/python/elastic_agent/torch/training.py

```diff
@@ -405,6 +409,9 @@ def __init__(
         self._save_ckpt_executor = ThreadPoolExecutor(max_workers=1)
         self._save_ckpt_future = None
         self._log_file = os.getenv(NodeEnv.TRAINING_LOG_FILE, "")
+        self._diagnose_agent = DiagnoseAgent(
```

Comment:
"error code is 507035" is internal logic, should not appear in the input parameters.